yolo/exporter.py

```diff
@@ -89,9 +92,11 @@ def export_blob(self):
             bin=str(self.f_bin.resolve()),#as_posix(),
             data_type="FP16",
             shaves=self.n_shaves,
-            version="2022.1",
+            version="2022.1" if self.use_rvc2 else "2022.3_RVC3",
             use_cache=False,
-            output_dir=self.conv_path.resolve()
+            output_dir=self.conv_path.resolve(),
+            url="https://blobconverter.luxonis.com" if self.use_rvc2 else \
```

Comment:
Would probably be good if this is read from ENV variable. We'll likely migrate dev-blobconverter to the main blobconverter soon. And ideally we don't need it hardcoded so it's an easy change.